lighthouse/constants.py

```diff
@@ -10,9 +10,14 @@
 FIELD_PLATE_BARCODE = "plate_barcode"
 FIELD_COG_BARCODE = "cog_barcode"
 FIELD_DATE_TESTED = "Date Tested"
+FIELD_CQ_1 = "ch1_cq"
+FIELD_CQ_2 = "ch2_cq"
+FIELD_CQ_3 = "ch3_cq"
 
 # MLWH lighthouse samples table field names
 MLWH_LH_SAMPLE_ROOT_SAMPLE_ID ="root_sample_id"
 MLWH_LH_SAMPLE_COG_UK_ID = "cog_uk_id"
 MLWH_LH_SAMPLE_RNA_ID = "rna_id"
-MLWH_LH_SAMPLE_RESULT = "result"
\ No newline at end of file
+MLWH_LH_SAMPLE_RESULT = "result"
+
+CT_VALUE_LIMIT = 30
```

Comment:
Unsure how datatypes work in python, we want a float as the values are e.g. 27.2631537